do not lowercase basename

sezero opened the pull request:
This reapplies the fix for https://github.com/adtools/sfdc/issues/3
which commit 2026f110a22 (https://github.com/adtools/sfdc/issues/8)
seems to have reverted without any specific explanation.

sezero commented:
PING?

sezero commented:
PING?

Futaura commented:
So, I've been looking at this and problem was that your fix was to `sfdc` only, which is generated by joining all the other `.pl` files together. Hence, you should have made the change to `main.pl` instead. That is why your change got wiped out.

sezero commented:
If you have a patch to that effect, it'd be welcome.

sezero committed:
do not lowercase basename

This reapplies the fix for https://github.com/adtools/sfdc/issues/3
which commit 2026f110a22 (https://github.com/adtools/sfdc/issues/8)
seems to have reverted.

sezero commented:
> If you have a patch to that effect, it'd be welcome.

Never mind, figured it out: change applied to main.pl in 7e7ac6ca and force-pushed.

Futaura reviewed (approved):
Thanks for this - hopefully this will now be permanent
